commonlib-build.properties

```diff
@@ -18,13 +18,14 @@ inject.jar=lib/guice-3.0/javax.inject.jar
 log4j.jar=lib/log4j/log4j-1.2.17.jar
 commonsio.jar=lib/commons-io-2.4/commons-io-2.4.jar
 derby.jar=lib/Derby/derby.jar
+influx.jar=lib/influxdb-java-2.5.jar
```

Comment:
It seems like if we go this route we need put all of the jars here (and move them into the commonlib/lib folder)